### ISSUE TITLE
Issue with `Imports must be ordered in lexicographic order`

### ISSUE THREAD

**Issue by DouglasCF** [JLLeitschuh/ktlint-gradle#585]
Hey! I'm using the ktlint on my Android project and I'm facing this issue: Imports must be ordered in lexicographic order.

This is an example of imports in lexicographic order using this [online tool](https://tools.knowledgewalls.com/online-lexicographic-ascending-or-descending-sort).

```
import io.mockk.every
import io.mockk.mockk
import kotlin.test.Test
import kotlin.test.assertEquals
import kotlinx.coroutines.ExperimentalCoroutinesApi
import kotlinx.coroutines.test.runTest
```

kltintChack says that this import is wrong, but actually it's correct.

So, always that I format my code on Android Studio it fixes the imports, but when running `ktlintCheck` it says it's wrong and format in a different way from expected.

Is this really a issue from the library side or I'm missing something?

**Comment by AleksanderBrzozowski** [comment]
Please provide more details - which version of the plugin you use?

**Comment by DmitriyZaitsev** [comment]
It seems, the tool doesn't like this import order
```
import kotlin.test.Test
import kotlin.test.assertEquals
```
and wants `assertEquals` to be before `Test`, even though 'a' > 'T'.

**Comment by wakingrufus** [comment]
This behavior is within ktlint itself, not the plugin, so this should be reported upstream to [ktlint](https://github.com/pinterest/ktlint)
